src/vocgui/migrations/0007_document_arcticle.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.1 on 2020-09-17 09:49
```

Comment:
Why is this migration file needed additional to the one below with the right `max_length`?